File: models/matchTimeModel.js
const mongoose=require('mongoose')
const Schema=mongoose.Schema

const MatchTimeSchema=new Schema({
    startTime:{//başlangıç saati
        type:String
    },
    stopTime:{//bitiş saati
        type:String
    },
    matchPrice:{//ilgili saat için maç ücreti
        type:Number,
        default:0.0
    },
    servicePrice:{//ilgili saat için servis ücreti
        type:Number,
        default:0.0
    },
    status:{//false ise ilgili saate maç rezervasyonu yapılamaz.
        type:Boolean,
        default:true
    }
},{collation:'matchTime'})

module.exports=mongoose.model('matchTime',MatchTimeSchema)

File: frontend/HaliSahaRezervasyonPortali/Models/matchTimeModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using Newtonsoft.Json;
using System.ComponentModel.DataAnnotations;

namespace HaliSahaRezervasyonPortali.Models
{
    public class matchTimeModel
    {
        [Display(Name = "ID")]
        [JsonProperty("_id")]
        public string _id { get; set; }

        [Display(Name = "Başlangıç Saati")]
        [JsonProperty("startTime")]
        public string startTime { get; set; }

        [Display(Name = "Bitiş Saati")]
        [JsonProperty("stopTime")]
        public string stopTime { get; set; }

        [Display(Name = "Maç Ücreti")]
        [JsonProperty("matchPrice")]
        public double matchPrice { get; set; }
        
        [Display(Name = "Servis Ücreti")]
        [JsonProperty("servicePrice")]
        public double servicePrice { get; set; }

        [Display(Name = "Durum")]
        [JsonProperty("status")]
        public Boolean status { get; set; }        
    }
}